plugins/cfg/config.yaml

```diff
@@ -115,6 +115,8 @@ plugins:
     topic-holds-puck: "~/RobotinoSim/GripperHasPuck/"
     topic-joint: "/GripperJoints/Holding"
     radius-grab-area: 0.05
+    prob-puck-falls: 0.10
```

Comment:
Short comment right above about the interval would be beneficial, as it's not immediately clear what this probability is.